pysite/migrations/tables/snake_facts/initial_data.json

```diff
@@ -0,0 +1,79 @@
+[
+   {"fact": "The decapitated head of a dead snake can still bite, even hours after death. These types of bites usually contain huge amounts of venom."},
```

Comment:
Can we have 4 spaces of indentation in this file, to remain consistent with `initial_data.json`?